src/scripts/choices.ts

```diff
@@ -1461,7 +1461,7 @@ class Choices implements Choices {
           the input was not focussed at the time of key press
           therefore does not have the value of the key.
         */
-        this.input.value += keyString.toLowerCase();
+        this.input.value += event.key.toLowerCase();
```

Comment:
Was this required for the non-Latin characters to display correctly?

Reply:
yes, Exactly.